pysteps/io/importers.py

```diff
@@ -135,6 +135,119 @@
     PYPROJ_IMPORTED = False
 
 
+def import_crri_eu(filename, **kwargs):
+    """Import a NetCDF radar rainfall product from the CRRI SAF.
+
+    Parameters
+    ----------
+    filename : str
+        Name of the file to import.
+
+    Returns
+    -------
+    out : tuple
+        A three-element tuple containing the rainfall field in mm/h imported
+        from the CRRI SAF netcdf, the quality field and the metadata. The
+        quality field is currently set to None.
+
+    """
+    if not NETCDF4_IMPORTED:
+        raise MissingOptionalDependency(
+            "netCDF4 package is required to import CRRI SAF products "
+            "but it is not installed"
+        )
+
+    R = _import_crri_eu_data(filename)
+
+    geodata = _import_crri_eu_geodata(filename)
+    metadata = geodata
+
+    # TODO(import_crri_eu): Add missing georeferencing data.
+
+    metadata["transform"] = None
+    metadata["zerovalue"] = np.nanmin(R)
+    if np.any(np.isfinite(R)):
+        metadata["threshold"] = np.nanmin(R[R > np.nanmin(R)])
+    else:
+        metadata["threshold"] = np.nan
+
+    # cut a window, to be defined here and only here
+    if True:
+        ny, nx = R.shape
+        nx1 = 440
+        nx2 = nx - 440
+        ny1 = 250
+        ny2 = ny
+        R = R[ny1:ny2, nx1:nx2]
+        x1, x2 = metadata['x1'], metadata['x2']
+        xpixelsize = metadata['xpixelsize']
+        x1n = x1 + xpixelsize * nx1
+        x2n = x1 + xpixelsize * (nx2 - 1)
+        metadata['x1'], metadata['x2'] = x1n, x2n
+        y1, y2 = metadata['y1'], metadata['y2']
+        ypixelsize = metadata['ypixelsize']
+        y1n = y2 - ypixelsize * (ny2 - 1)
+        y2n = y2 - ypixelsize * ny1
+        metadata['y1'], metadata['y2'] = y1n, y2n
+
+    return R, None, metadata
+
+
+def _import_crri_eu_data(filename):
+    ds_rainfall = netCDF4.Dataset(filename)
+    if "crr_intensity" in ds_rainfall.variables.keys():
+        data = np.array(ds_rainfall.variables["crr_intensity"])
+        precipitation = np.where(data == 65535, np.nan, data)
+    else:
+        precipitation = None
+    ds_rainfall.close()
+
+    return precipitation
+
+
+def _import_crri_eu_geodata(filename):
+
+    geodata = {}
+
+    ds_rainfall = netCDF4.Dataset(filename)
+
+    # get projection
+    projdef = ds_rainfall.getncattr('gdal_projection')
+    geodata["projection"] = projdef
+
+    # get x1, y1, x2, y2, xpixelsize, ypixelsize, yorigin
+    px = ds_rainfall.variables['nx'][:]
+    py = ds_rainfall.variables['ny'][:]
+    geotable = ds_rainfall.getncattr('gdal_geotransform_table')
+    xmin = px.min()
+    xmax = px.max()
+    ymin = py.min()
+    ymax = py.max()
+    xpixelsize = abs(geotable[1])
+    ypixelsize = abs(geotable[5])
+    factor_scale = 1.0
```

Comment:
Is this 'factor_scale' a changeble factor? If so, I would recommend to add it as e.g. '**kwargs' in the function (unless it's already somewhere defined and I've missed it..). If not, we won't need this variable at all, I assume.

Reply:
removed in cd52933